src/main/java/jp/co/yahoo/yosegi/hadoop/tools/sort/Sort.java

```diff
@@ -0,0 +1,271 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jp.co.yahoo.yosegi.hadoop.tools.sort;
+
+import jp.co.yahoo.yosegi.hadoop.mapreduce.YosegiParserOutputFormat;
+import jp.co.yahoo.yosegi.hadoop.mapreduce.YosegiSpreadInputFormat;
+import jp.co.yahoo.yosegi.hadoop.mapreduce.YosegiStatsInputFormat;
+import jp.co.yahoo.yosegi.message.parser.IParser;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.GnuParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.OptionBuilder;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.conf.Configured;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapreduce.Job;
+import org.apache.hadoop.mapreduce.lib.output.NullOutputFormat;
+import org.apache.hadoop.util.Tool;
+import org.apache.hadoop.util.ToolRunner;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+public class Sort extends Configured implements Tool {
+
+  private static final long FILE_SIZE = 1024 * 1024 * 128;
```

Comment:
File size is constant? If so, could you please the reason why this "FILE_SIZE" can treat as constant.
Any way, this matter is not critical, then it requires to modify.

Reply:
Hadoop's block size is often 64/128/256 MB.
So I made it 128MB.

Reply:
I think as following comment is helpful for code reader. 
private static final long FILE_SIZE = 1024 * 1024 * 128; // from Hadoop's block size

Reply:
I agree. I added it.